mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
Ok, url_link makes more sense.  You don't think this view is at all related to the existing url view?

What should that view give on other types?  Seems illogical for it to be a link at all, which is why I think it should come back as core or raw view (core seems right, but I'm not sure it matters).  We just want to see something when this view is used on other types, otherwise debugging is harder.